estuary_updater/handlers/base.py

```diff
@@ -105,9 +123,14 @@ def get_or_create_build(self, identifier, original_nvr=None, force_container_lab
             log.error('Failed to get brew build using the identifier {0}'.format(identifier))
             raise
 
+        try:
+            extra_json = json.dumps(build_info['extra'])
+        except (ValueError, TypeError):
```

Comment:
When would this ever happen?

Reply:
This was how it was written in the scrapers, but I can change it there too, or is it a different case there?